CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Changelog
+
+## [Unreleased]
+
+### Changed
+* Refactor test options to always set `createIfMissing` and `errorIfExists` (@ralphtheninja)
+* Move `setUp()` function into `test()` (@ralphtheninja)
+* Move `openTestDatabase()` calls into `test()` and pass `ctx` to tests (@ralphtheninja)
+* Test error after `db.close()` and after `cleanup()` (@ralphtheninja)
+* Use `after` in `cleanup()` (@ralphtheninja)
+* Use only `readable-stream` from user land (@ralphtheninja)
+* Use `^` for devDependencies (@ralphtheninja)
+* Switch to plain MIT license (@ralphtheninja)
+* Replace `util.inherits` with `inherits` module (@ralphtheninja)
+* Replace `this._destroyed` with `this.destroyed` from `Writable` (@ralphtheninja)
+* Export single function that creates the stream (@ralphtheninja)
+* Flip parameters in `WriteStream` constructor (@ralphtheninja)
+* Verify results once using `level-concat-iterator` intead of multiple `db.get()` operations (@ralphtheninja)
+* Update README style (@ralphtheninja)
+
+### Added
+* Add node 6, 8, 9 and 10 to Travis (@ralphtheninja)
+* Add `standard` for linting (@ralphtheninja)
+* Test race condition (@vweevers)
+* Add `nyc` and `coveralls` (@vweevers)
+
+### Removed
+* Remove node 0.10, 2, 3, 4 and 5 from Travis (@ralphtheninja)
+* Remove `contributors` from `package.json` (@ralphtheninja)
+* Remove copyright headers from code (@ralphtheninja)
+* Remove `this.{writable,readable}` state (@ralphtheninja)
+* Remove `this._db.isOpen()` checks (@ralphtheninja)
+* Remove patching db from the API (@ralphtheninja)
+* Remove encoding options (@ralphtheninja, @vweevers)
+
+### Fixed
+* Fix erroneous test on missing type (@ralphtheninja)
+* Fix race condition by flushing before finish (@vweevers)
+* Fix `coveralls` by removing `COVERALLS_PARALLEL` and webhook (@vweevers)
```

Comment:
Can leave this out (it's part of "Add `nyc` and `coveralls`")

Reply:
I was thinking the same.